package.json

```diff
@@ -47,19 +47,19 @@
     "systelab-preferences": "^8.0.0",
```

Comment:
We need to publish the systelab-preferences, systelab-charts, systelab-translate to merge this PR